molecule.yml

```diff
@@ -0,0 +1,22 @@
+---
+ansible:
+  config_file: ansible.cfg
```

Comment:
I think the `requirements_file` setting is missing, along with the associated `requirements.yml` file that includes `azavea.java`.